dubbo-remoting/dubbo-remoting-http12/src/main/java/org/apache/dubbo/remoting/http12/exception/HttpOverPayloadException.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.dubbo.remoting.http12.exception;
+
+public class HttpOverPayloadException extends HttpStatusException {
+
+    public HttpOverPayloadException(String message) {
+        super(500, message);
```

Comment:
Should be 413 here.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/413

Reply:
4xx is a client error, 5xx is a server error, and response too large is a server error, I don't think 413 can be used.

Reply:
Yes, for request body we can yhou 413, respone should use 500